message tab_FogChamberLevelAttrTable 
{
	optional string tname = 1;
	repeated tab_FogChamberLevelAttr tlist = 2;
}

message tab_FogChamberLevelAttr
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 Level = 3;
	optional int32 FogHeight = 4;
	optional int32 showTime = 5;
}